app-typescript/components/TagInput.tsx

```diff
@@ -2,21 +2,13 @@ import * as React from 'react';
 import { Chips } from '@superdesk/primereact/chips';
 import {InputWrapper} from './Form';
 import nextId from "react-id-generator";
+import {IInputWrapper} from './Form/InputWrapper';
+import {PillPreview} from './PreviewPill';
 
-interface IProps {
+interface IProps extends IInputWrapper {
     value: Array<string>;
     onChange(value: Array<string>): void;
-    placeholder: string;
-    invalid?: boolean;
-    inlineLabel?: boolean;
-    labelHidden?: boolean;
-    tabindex?: number;
-    fullWidth?: boolean;
-    info?: string;
-    error?: string;
-    required?: boolean;
-    label?: string;
-    disabled?: boolean;
+    placeholder?: string;
```

Comment:
Shouldn't placeholder be in `IInputWrapper` since every input might need a placeholder value

Reply:
not all inputs support placeholders, for example `select` doesn't